Tak.py

```diff
@@ -1,21 +1,164 @@
 from collections import deque, namedtuple
+from enum import Enum
+from logging import raiseExceptions
 
 GameState = namedtuple('GameState', 'to_move, board, pieces, moves')
 
+class pieceType(Enum):
+    """"Enum enables easy access to standardized piece characteristics"""
+    WALL,TILE,CAPSTONE = range(3)
+class pieceColor(Enum):
```

Comment:
Small thing: shouldn't there be a newline between lines 9 and 10? And updated comments for the enums.